documentContentProviders/dockerInspect.ts

```diff
@@ -0,0 +1,22 @@
+import { TextDocumentContentProvider, Uri, window, workspace } from "vscode";
+import { docker } from "../commands/utils/docker-endpoint";
+
+export const IMAGE_DOMAIN = "image";
+export const SCHEME = "docker-inspect";
+export const URI_EXTENSION = ".json";
```

Comment:
@lostintangent what was the motivation behind removing the `-watch` from `package.json`?